rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/parametric/ParametricTextDocumentService.java

```diff
@@ -227,7 +221,8 @@ public void connect(LanguageClient client) {
     @Override
     public void didOpen(DidOpenTextDocumentParams params) {
         logger.debug("Did Open file: {}", params.getTextDocument());
-        handleParsingErrors(open(params.getTextDocument()));
+        TextDocumentState file = open(params.getTextDocument());
+        handleParsingErrors(file, file.getCurrentDiagnosticsAsync()); // No debounce
```

Comment:
what if we introduce 2 globals in this class?

```java
private static final Duration NO_DEBOUNCE = Duration.ZERO;
private static final Duration NORMAL_DEBOUNCE = Duration.ofMillis(800);
```

And we use them in the 2 calls? so that we don't need this comment?

(and share them in some way with `RascalTextDocumentService.java`)